src/js/components/AppListComponent.jsx

```diff
@@ -168,7 +171,9 @@ var AppListComponent = React.createClass({
   },
 
   hasFilters: function () {
-    return Object.values(this.props.filters).some(filter => {
+    var filters = Object.assign(this.getQueryParamObject(), this.props.filters);
```

Comment:
Why are you extending with props.filters ?
Actually props.fitlers could be removed completely, couldn't it?